pkg/configuration/configuration.go

```diff
@@ -51,7 +54,7 @@ type Config struct {
 	UpdateIntervalSeconds                    int64    `json:"updateIntervalSeconds,omitempty"`
 	DefaultDecisionSeconds                   int64    `json:"defaultDecisionSeconds,omitempty"`
 	HTTPTimeoutSeconds                       int64    `json:"httpTimeoutSeconds,omitempty"`
-	ForwardedHeadersCustomName               string   `json:"forwardedheaderscustomheader,omitempty"`
+	ForwardedHeadersCustomName               string   `json:"forwardedHeadersCustomHeader,omitempty"`
```

Comment:
The variable naming here is no logical,
we should use 
 	ForwardedHeadersCustomName               string   `json:"forwardedheaderscustomName,omitempty"`

Reply:
I will make another PR to change that, it need to appear in the changelog